pkg/pkg.bzl

```diff
@@ -503,68 +504,171 @@ def pkg_tar(name, **kwargs):
+See https://www.debian.org/doc/debian-policy/ch-files.html#s-config-files.""",
+            allow_single_file = True,
+        ),
+        "priority": attr.string(
+            doc = """The priority of the package.
+            See http://www.debian.org/doc/debian-policy/ch-archive.html#s-priorities.""",
+        ),
+        "section": attr.string(
+            doc = """The section of the package.
+            See http://www.debian.org/doc/debian-policy/ch-archive.html#s-subsections.""",
+        ),
+        "homepage": attr.string(doc = """The homepage of the project."""),
+
+        "breaks": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
+        "conflicts": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
+        "depends": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
+        "depends_file": attr.label(
+            doc = """File that contains a list of package dependencies. Must not be used with `depends`.
+            See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            allow_single_file = True,
+        ),
+        "enhances": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
+        "provides": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
+        "predepends": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
+        "recommends": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
+        "replaces": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
+        "suggests": attr.string_list(
+            doc = """See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.""",
+            default = [],
+        ),
 
         # Common attributes
-        "out": attr.output(mandatory = True),
-        "package_file_name": attr.string(doc = "See Common Attributes"),
+        "out": attr.output(
+            doc = """See Common Attributes""",
+            mandatory = True
+        ),
+        "package_file_name": attr.string(
+            doc = """See Common Attributes.
+            Default: "%{package}-%{version}-%{architecture}.deb""",
```

Comment:
The [reference](https://github.com/bazelbuild/rules_pkg/blob/main/pkg/docs/reference.md#common-attributes) suggests that the `%`'s shouldn't be here, perhaps.

Reply:
Removed